spec/spec.md

```diff
@@ -751,14 +764,16 @@ experience for users.
 
 - The message object ****MUST**** contain a `descriptor` property, and its value ****MUST**** be a JSON object composed as follows:
   - The object ****MUST**** contain a `method` property, and its value ****MUST**** be the string `CollectionsQuery`.
-  - The object ****MAY**** contain a `schema` property, and if present its value ****Must**** be a URI string that indicates the schema of the associated data.
-  - The object ****MAY**** contain an `recordId` property, and its value ****MUST**** be a [[spec:rfc4122]] UUID Version 4 string.
-  - The object ****MAY**** contain a `dataFormat` property, and its value ****MUST**** be a string that indicates the format of the data in accordance with its MIME type designation. The most common format is JSON, which is indicated by setting the value of the `dataFormat` property to `application/json`.
-  - The object ****MAY**** contain a `dateSort` field, and if present its value ****MUST**** be one of the following strings:
-      - `createdAscending`: return results in order from the earliest `dateCreated` value to the latest.
-      - `createdDescending`: return results in order from the latest `dateCreated` value to the earliest.
-      - `publishedAscending`: return results in order from the earliest `datePublished` value to the latest.
-      - `publishedDescending`: return results in order from the latest `datePublished` value to the earliest.
+  - The object ****MAY**** contain a `filter` property, and if present its value ****MUST**** be an object that ****MAY**** contain the following properties:
+    - The object ****MAY**** contain a `contextId` property, and its value ****MUST**** be a [[spec:rfc4122]] UUID Version 4 string.
```

Comment:
Another global search-and-replace, from `a [[spec:rfc4122]]` to `an [[spec:rfc4122]]`